README.md

```diff
@@ -96,7 +96,7 @@ installation of the following files:
     sudo cp ratbagd.service /etc/systemd/system/ratbagd.service
 
 The files are installed into the prefix by make install, see also the
```

Comment:
Done. I pushed a regular commit as I changed some formatting and added a reference as well.